packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -265,7 +265,7 @@ const ToggleMenuDrawer = React.forwardRef(
                         <MobileDrawer.Item key={idx}>
                             <MenuLink
                                 is_language
-                                is_active={currentLanguage === lang}
+                                is_active={localStorage.getItem('i18n_language') === lang}
```

Comment:
What advantage does it provide to directly call localStorage here instead of using our function? What if the localStorage is empty?

Reply:
Our getLanguage function was run only on the initial render of the component and the active language was not getting updated and the language that we logged in with was shown as the current language.
One of the ways to implement it was to check the active language with the one that is set in the localStorage which gets updated after any changes of the languages.
Another way to implement it is to call the getLanguage function in the getLanguageRoutes function to update our current language.
Let me stick to the current implementation of the component and just fix the issue with where our current language does not update on our language changes.

Reply:
We were using the current language of the language that we logged in and it was not getting updated after our changes on the languages. Considering that I used localStorage since it gets updated on language changes and it would updated the current language that we are using and would highlight the correct one.
I'm gonna stick to the current implementation that we have and just update the current language by using getLanguage function from translations and call it in getLanguageRoutes to have the expected behavior for our active language.